README.md

```diff
@@ -1042,6 +1043,22 @@ is always valid for 90 days and will be rolled 30 days before it expires by upda
 in the `Certificate` object.  
 The configuration can be changed with the command line parameter `--issuer.renewal-window`.
 
+### Triggering a manual Certificate renewal
+
+You can trigger a manual renewal of a `Certificate` by setting `.spec.renew` to `true`.
+The controller will then renew the certificate with the next reconciliation and remove the field.
+
+```yaml
+apiVersion: cert.gardener.cloud/v1alpha1
+kind: Certificate
+metadata:
+  name: renew-sample
+  namespace: default
+spec:
+  commonName: cert1.mydomain.com
+  renew: true # trigger a renewal with the next reconciliation, the field will be removed
```

Comment:
```suggestion
  renew: true # trigger a renewal with the next reconciliation, the field will be removed
  ensureRenewedAfter: null # mandatory if a manual renewal was already triggered 
```

Reply:
For repeatable renewal, one need to deal with the field `ensureRenewedAfter`. It's not nice from user perspective, but at the moment it is as it is.